.github/workflows/contracts.yml

```diff
@@ -40,5 +40,6 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - uses: ./.github/actions/setup-contracts-clients
-    - uses: ./.github/actions/local-contract-deploy
+    - uses: ./.github/actions/local-contract-deploy-hardhat
```

Comment:
I decided to separate the contract deploy scripts out into a hardhat version and a geth version, as using just the geth version appeared to cause issues with `test-gas-measurements`. See [this failing workflow](https://github.com/web3well/bls-wallet/actions/runs/4142899084/jobs/7164203145). This was resolved by running these measurements against hardhat. Imagine the issue with using geth was line 45 in this file.

Reply:
You can observe the error locally by running `yarn start` and then `yarn hardhat run ./scripts/measure_gas/run.ts --network gethDev`